reactivexcomponent/reactivexcomponent/communication/xc_session.py

```diff
@@ -1,12 +1,16 @@
 import ssl
 import websocket as WebSocket
 from reactivexcomponent.communication.publisher import Publisher
+from reactivexcomponent.configuration.api_configuration import APIConfiguration
 
 SUCCESS = None
+
+
 class XcSession:
 
     def __init__(self):
         self.websocket = None
+        self.configuration = APIConfiguration
```

Comment:
???

shoudn't it be an instance of APIConfiguration?

like 

`self.configuration = APIConfiguration()`